perfil/mandate/models.py

```diff
@@ -5,10 +5,13 @@
 
 
 class Politician(models.Model):
-    congressperson_id = models.CharField(max_length=250)
+    congressperson_id = models.CharField(max_length=250, unique=True)
```

Comment:
I think this might be a problem. AFAIK `congressperson_id` we use comes from the Chamber of Deputy, but will not match the number/ID from other institutions (Senate, for example).

Reply:
I thought this was a problem, but not for now. I would like to merge this and fix this on the next PR so we won't have a migrations problem

Reply:
Let's just add a `  # TODO` comment to copy that?

Reply:
Will do